assets/main.css

```diff
@@ -279,7 +279,19 @@ a {
 }
 
 input[type='file'] {
-  display: none;
+  width: 0.1px;
+  height: 0.1px;
```

Comment:
the width and height rules here don't seem necessary since it's `display: absolute` and `opacity: 0`

Reply:
It doesn't seem important to me, but just in case, those rules were responsible for the 'hidden' form input fitting behind the label used as a button

![browse_size](https://user-images.githubusercontent.com/8090731/31053978-5bc1a396-a6a0-11e7-9600-ba59b665ac12.png)